src/index.ts

```diff
@@ -35,17 +35,7 @@ window.Spotfire.initialize(async (mod) => {
         const hasColorExpression = !!colorAxis.parts.length && colorAxis.isCategorical;
         const { tooltip } = mod.controls;
 
-        let colorAxisCategoryName = hasColorExpression ? colorAxis.parts[0].displayName : null,

```

Comment:
Were these variables not used before?

Reply:
They were used before to create the text for the tooltip, but since that is handled by the API now it should not be needed